MAINTAINERS.md

```diff
@@ -1,7 +1,15 @@
-# Helm Maintainers
+# Helm Classic Maintainers
 
-This document serves to describe the leadership structure of the Helm project, and to list the current
-project maintainers.
+|![](https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/Warning.svg/156px-Warning.svg.png) | Helm Classic (the `helm/helm` repository) is **no longer actively developed** but will remain available until `kubernetes/helm` has stabilized.
+|---|---|
+
+Helm and [Deployment Manager](https://github.com/kubernetes/deployment-manager)
+have recently joined forces to make deploying and managing software on
+Kubernetes as simple as possible. The combined effort now lives in the Kubernetes GitHub organization at
+[kubernetes/helm](https://github.com/kubernetes/helm).
+
+This document serves to describe the leadership structure of the Helm Classic project, and to list the current
+project maintainers. _Maintainers of Helm Classic currently perform bug fixes and critical maintenance only._
 
 # What is a maintainer?
```

Comment:
I think "what" is appropriate here since what follows is a definition of the role rather than a list of who currently fulfills the role.